feed/views.py

```diff
@@ -1217,21 +1185,19 @@ def list(self, request):
 class MilestoneViewSet(viewsets.ModelViewSet):
 
     def list(self, request):
-        if request.user.is_superuser:
-            queryset = self.queryset
-        else:
+        # Use this queryset or the django-filters lib will not work
```

Comment:
I think this comment can be removed from every function. You are a mature and trusted developer, and if I doubted why you are using the filter_queryset function then I use the PR review to ask you.

Reply:
I also think I could remove the comments but the problem is: when another developer gets the code and changes it back as it was before. I wanna know from you: Is a short warning comment in the code useless?

Reply:
With "this queryset" you mean `self.get_queryset()`?

Reply:
No. I meant the whole line: `queryset = self.filter_queryset(self.get_queryset())`